### ISSUE TITLE
Crash when opening images, caused by android.os.TransactionTooLargeException

### ISSUE THREAD

**Issue by TranceLove** [TeamAmaze/AmazeFileManager#1482]
Reported by an user using Nokia 6 (TA-1000) running Android 8.1.0 and using FOTO Gallery to open image files.

Steps to reproduce:
1. Open Amaze
2. Open images under Quick access category
3. Open an image
4. Amaze crash

Interesting messages in logcat (though not very useful IMO)
```
10-22 17:57:50.650  6950  6950 E JavaBinder: !!! FAILED BINDER TRANSACTION !!!  (parcel size = 5942452)
10-22 17:57:50.651  6950  6950 W ActivityThread: Bundle stats:
10-22 17:57:50.652  6950  6950 W ActivityThread:   android:viewHierarchyState [size=5784]
10-22 17:57:50.653  6950  6950 W ActivityThread:     android:views [size=5680]
10-22 17:57:50.653  6950  6950 W ActivityThread:       0x7f090111 [size=2492]
10-22 17:57:50.875  6950  6950 W ActivityThread:   android:support:fragments [size=5935948]
10-22 17:57:50.875  6950  6950 W ActivityThread: PersistableBundle stats:
10-22 17:57:50.875  6950  6950 W ActivityThread:   [null]
10-22 17:57:50.875  6950  6950 D AndroidRuntime: Shutting down VM
--------- beginning of crash
10-22 17:57:50.876  6950  6950 E AndroidRuntime: FATAL EXCEPTION: main
10-22 17:57:50.876  6950  6950 E AndroidRuntime: Process: com.amaze.filemanager, PID: 6950
10-22 17:57:50.876  6950  6950 E AndroidRuntime: java.lang.RuntimeException: android.os.TransactionTooLargeException: data parcel size 5942452 bytes
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at android.app.ActivityThread$StopInfo.run(ActivityThread.java:3950)
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:790)
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:99)
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:164)
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6501)
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
10-22 17:57:50.876  6950  6950 E AndroidRuntime: Caused by: android.os.TransactionTooLargeException: data parcel size 5942452 bytes
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at android.os.BinderProxy.transactNative(Native Method)
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at android.os.BinderProxy.transact(Binder.java:764)
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at android.app.IActivityManager$Stub$Proxy.activityStopped(IActivityManager.java:4692)
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        at android.app.ActivityThread$StopInfo.run(ActivityThread.java:3934)
10-22 17:57:50.876  6950  6950 E AndroidRuntime:        ... 7 more
10-22 17:57:50.895  1540  1556 I ActivityManager: Showing crash dialog for package com.amaze.filemanager u0
```

[logcat.txt](https://github.com/TeamAmaze/AmazeFileManager/files/2509188/logcat.txt)

**Comment by Gerii** [comment]
I have the same issue on Amaze >v3.3 with a folder that has a lot of videos (>500 files). The application always crashes when it goes to the background while that folder is open (e.g., by opening a file as the OP suggested, or just by clicking the home button while that folder is open). It seems that the state that Amaze saves for later is too big, as described here:
https://medium.com/@mdmasudparvez/android-os-transactiontoolargeexception-on-nougat-solved-3b6e30597345

**Comment by VishalNehra** [comment]
This should be fixed on 3.5.x
